spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,72 @@
+feature 'FEATURE: sign up' do
+
+  scenario 'Can click to sign up from home page' do
+    visit('/')
+    click_link('sign_up')
+    expect(page.current_path).to eq "/users/new"
+  end
+
+  let(:name) { 'J T Kirk' }
+  let(:email) { 'JTK@test.com' }
+  let(:user_name) { 'TheShat' }
+  let(:password) { 'password' }
+  let(:password_confirmation) { 'password' }
```

Comment:
These have been repeated a few times — *might* be worth extracting them someplace common, though some might say that makes them a [mystery guest](https://robots.thoughtbot.com/mystery-guest).